terragrunt/modules/dns-zone/outputs.tf

```diff
@@ -0,0 +1,9 @@
+output "name" {
```

Comment:
To avoid too many files, I'd move the outputs (and the inputs) into `_terraform.tf`.

Reply:
The [Standard Module Structure](https://developer.hashicorp.com/terraform/language/modules/develop/structure) for Terraform modules explicitly names `main.tf`, `variables.tf`, and `outputs.tf`. For stuff in `modules/shared`, I think it's helpful to follow that convention.